mega-zep-backend/src/main/java/com/gepardec/mega/rest/ProjectCommentResource.java

```diff
@@ -0,0 +1,43 @@
+package com.gepardec.mega.rest;
+
+import com.gepardec.mega.application.interceptor.Secured;
+import com.gepardec.mega.rest.model.ProjectCommentDto;
+import com.gepardec.mega.service.api.project.ProjectCommentService;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@Secured
+@RequestScoped
+@Path("/projectcomments")
+public class ProjectCommentResource {
+
+    @Inject
+    ProjectCommentService projectCommentService;
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public ProjectCommentDto getProjectComment(
```

Comment:
We are in the ProjectCommentRessource, therefore only get, create and update instead of getProjectComment, etc. -> same in service